Src/Common/CrossComponentCorrelationEventSource.cs

```diff
@@ -15,12 +15,12 @@
     /// <summary>
     /// ETW EventSource tracing class.
     /// </summary>
-    [EventSource(Name = "Microsoft-ApplicationInsights-Extensibility-CrossComponentCorrelation")]
-    internal sealed class CrossComponentCorrelationEventSource : EventSource
+    [EventSource(Name = "Microsoft-ApplicationInsights-Extensibility-AppMapCorrelation")]
+    internal sealed class AppMapCorrelationEventSource : EventSource
```

Comment:
Because it's not just cross component correlation related anymore. It also deals with retrieving headers for correlating services within the same AI resource (so I just chose a generic name app map correlation).

I'm not sure which list in functional tests you are talking about. Can you please point me to it.